kogito-benchmarks-framework/src/main/java/org/kie/kogito/benchmarks/framework/Commands.java

```diff
@@ -0,0 +1,484 @@
+    private static final Pattern numPattern = Pattern.compile("[ \t]*[0-9]+[ \t]*");
+    private static final Pattern quarkusVersionPattern = Pattern.compile("[ \t]*<quarkus.version>([^<]*)</quarkus.version>.*");
+    private static final Pattern trailingSlash = Pattern.compile("/+$");
+
+    public static String getArtifactGeneBaseDir() {
+        for (String p : new String[] { "ARTIFACT_GENERATOR_WORKSPACE", "artifact.generator.workspace" }) {
+            String env = System.getenv().get(p);
+            if (StringUtils.isNotBlank(env)) {
+                return env;
+            }
+            String sys = System.getProperty(p);
+            if (StringUtils.isNotBlank(sys)) {
+                return sys;
+            }
+        }
+        return System.getProperty("java.io.tmpdir");
+    }
+
+    public static String getLocalMavenRepoDir() {
+        for (String p : new String[] { "TESTS_MAVEN_REPO_LOCAL", "tests.maven.repo.local" }) {
+            String env = System.getenv().get(p);
+            if (StringUtils.isNotBlank(env)) {
+                return env;
+            }
+            String sys = System.getProperty(p);
+            if (StringUtils.isNotBlank(sys)) {
+                return sys;
+            }
+        }
+
+        String mainBuildRepo = System.getProperty("maven.repo.local");
+        if (StringUtils.isNotBlank(mainBuildRepo)) {
+            return mainBuildRepo;
+        }
+
+        return System.getProperty("user.home") + File.separator + ".m2" + File.separator + "repository";
+    }
+
+    /**
+     * Get system properties starting with `quarkus.native` prefix, for example quarkus.native.builder-image
+     * 
+     * @return List of `-Dquarkus.native.xyz=foo` strings
+     */
+    public static List<String> getQuarkusNativeProperties() {
+        List<String> quarkusNativeProperties = System.getProperties().entrySet().stream()
+                .filter(e -> e.getKey().toString().contains("quarkus.native"))
+                .map(e -> "-D" + e.getKey() + "=" + e.getValue())
+                .collect(Collectors.toList());
+        return quarkusNativeProperties;
+    }
+
+    public static String getQuarkusPlatformVersion() {
+        for (String p : new String[] { "QUARKUS_PLATFORM_VERSION", "quarkus.platform.version" }) {
+            String env = System.getenv().get(p);
+            if (StringUtils.isNotBlank(env)) {
+                return env;
+            }
+            String sys = System.getProperty(p);
+            if (StringUtils.isNotBlank(sys)) {
+                return sys;
+            }
+        }
+        logger.warn("Failed to detect quarkus.platform.version/QUARKUS_PLATFORM_VERSION, defaulting to getQuarkusVersion().");
+        return getQuarkusVersion();
+    }
+
+    public static String getQuarkusVersion() {
+        for (String p : new String[] { "QUARKUS_VERSION", "quarkus.version" }) {
+            String env = System.getenv().get(p);
```

Comment:
`getSystemPropertyOrEnvVarValue` method is not used here due to different order of checking?

Reply:
Probably, this was intentionally left from the Quarkus framework.